File: MainApp/src/main/java/com/sleticalboy/dailywork/bean/AudioItem.java
package com.sleticalboy.dailywork.bean;

import androidx.annotation.NonNull;

/**
 * Created on 20-9-3.
 *
 * @author Ben binli@grandstream.cn
 */
public class AudioItem {

    public String mTitle;
    public String mSummary;

    public int mColor;
    public int mColor2;

    @NonNull
    @Override
    public String toString() {
        return "AudioItem{" +
                "mTitle='" + mTitle + '\'' +
                ", mSummary='" + mSummary + '\'' +
                ", mColor=" + mColor +
                ", mColor2=" + mColor2 +
                '}';
    }
}

File: MainApp/src/main/java/com/sleticalboy/learning/bean/AudioItem.kt
package com.sleticalboy.learning.bean

/**
 * Created on 20-9-3.
 *
 * @author Ben binli@grandstream.cn
 */
class AudioItem {
  var mTitle: String? = null
  var mSummary: String? = null
  var mColor = 0
  var mColor2 = 0
  override fun toString(): String {
    return "AudioItem{" +
        "mTitle='" + mTitle + '\'' +
        ", mSummary='" + mSummary + '\'' +
        ", mColor=" + mColor +
        ", mColor2=" + mColor2 +
        '}'
  }
}